### ISSUE TITLE
Question: is it possible to load a docker image at build time?

### ISSUE THREAD

**Issue by fsamir** [hypriot/os-rootfs#48]
Hi all,

I tried both "docker pull" and "docker load", to build an image with the container I need, but  qemu seems to fail on running go (error below). 
Also, a quick google search showed several issues relating the problem.

Was anyone able to achieve this in any other way? 

```
fatal error: rt_sigaction failure

runtime stack:
runtime.throw(0x75e554, 0x14)
        /usr/local/go/src/runtime/panic.go:566 +0x78
```

Thanks,
Franklin

**Comment by StefanScherer** [comment]
Hi @fsamir 
I also tried that a week ago, but I also couldn't make it work. In HCL (Hypriot Cluster Lab) we used the "docker load" on first boot on the real device.l
Maybe running a dockerd without qemu using the rootfs with the correct storage driver could work to just do the "docker pull" to fill the filesystem. But this won't play nicely with multiarch docker images as then the Intel version will be pulled.
Cheers,
Stefan
